include/mxnet/operator.h

```diff
@@ -491,6 +491,9 @@ using FCompute = std::function<void (const nnvm::NodeAttrs& attrs,
                                      const std::vector<OpReqType> &req,
                                      const std::vector<TBlob> &outputs)>;
 
+using FTempResources = std::function<std::vector<ResourceRequest> (
```

Comment:
Can't find such file?

On 8/23/16, Tianqi Chen <notifications@github.com> wrote:
>> @@ -491,6 +491,9 @@ using FCompute = std::function<void (const
>> nnvm::NodeAttrs& attrs,
>>                                       const std::vector<OpReqType> &req,
>>                                       const std::vector<TBlob>
>> &outputs)>;
>>
>> +using FTempResources = std::function<std::vector<ResourceRequest> (
>
> see my repo, include/mxnet/op_attr_types.h
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/tqchen/mxnet/pull/2/files/d35adf3c7071a168d554252a7b73db5236672fef..729528512bb688d38ade2f635cfe3a86b38e9dfb#r75717294